src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -104,16 +106,49 @@ NativeAOT binaries built with this feature can run even when OpenSSL libraries a
 You can use this feature by adding the `StaticOpenSslLinking` property to your project file as follows:
 
 ```xml
-    <PropertyGroup>
-      <StaticOpenSslLinking>true</StaticOpenSslLinking>
-    </PropertyGroup>
+<PropertyGroup>
+  <StaticOpenSslLinking>true</StaticOpenSslLinking>
+</PropertyGroup>
 ```
 
 This feature is only supported on Linux. This feature is not supported when crosscompiling.
 
+License (Apache v2.0): https://github.com/openssl/openssl/blob/master/LICENSE.txt
```

Comment:
Nit: This applies to OpenSSL 3+ only. Earlier OpenSSL versions have different license.

Reply:
```suggestion
OpenSSL 3+ License (Apache v2.0): https://github.com/openssl/openssl/blob/master/LICENSE.txt
OpenSSL releases prior to v3 (dual OpenSSL and SSLeay license): https://www.openssl.org/source/license-openssl-ssleay.txt
```